packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -279,15 +279,10 @@ export class PrismicRepositoryManager extends BaseManager {
 										model,
 									});
 
-								await this.slices.updateSlice({
-									libraryID: change.libraryID,
-									model: modelWithScreenshots,
-								});
```

Comment:
I don't think this can be removed. Otherwise, from my understanding, the slice will stay on "changed" status because its model image URL will always be different from the local screenshot.